shared/desktop/remote/sync-avatar-props.desktop.js

```diff
@@ -31,10 +31,10 @@ function SyncAvatarProps(ComposedComponent: any) {
       if (windowComponent === this.props.windowComponent && windowParam === this.props.windowParam) {
         if (action.type === ConfigGen.loadAvatars) {
           const {usernames} = action.payload
-          this.props.setUsernames(this.props.usernames.add(usernames))
+          this.props.setUsernames(this.props.usernames.concat(usernames))
```

Comment:
noticed this while using react devtools. Using I.Set().add([1,2,3]) adds an array so set.size === 1 :(